sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,254 @@
+# March'23 release notes
+
+Release notes for commit range [`ca54ea30..cb91c232`](https://github.com/intel/llvm/compare/ca54ea30...cb91c232)
+
+## New features
+
+### SYCL Compiler
+- Added semi-dynamic SLM allocation - `sycl::ext::intel::experimental::esimd::slm_allocator`
+[e7674019]
+- Added a support of compilation of the `.cu` files containing CUDA, and SYCL 
+codes in the same translation unit [31608c24] [f5126d28]
+- Added a support of `amd-gpu-gfx1034` as an acceptable value for 
+`-fsycl-targets` [5e86a41d]
+
+### SYCL Library
+- Implemented [`sycl_ext_oneapi_matrix`](doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc) extension using new unified interface [f4a9ef1b]
+- Partially implemented [`sycl_ext_codeplay_kernel_fusion`](doc/extensions/experimental/sycl_ext_codeplay_kernel_fusion.asciidoc) extension for Linux: Intel CPU and GPU devices are fully supported, except specialization constants, streams, images and reductions [89e82e3f] 
+[5d7e86d4] [ab0c46bd] [b5d5aada] [325bc4e2] [cf245f62]
+- Added default constructor to `sycl::local_accessor` [589824d7]
+- Added support of `SYCL_PI_HIP_MAX_LOCAL_MEM_SIZE` environment variable for 
+HIP backend [92f6d688]
+- Aligned `sycl::vec` aliases with SYCL 2020 [1299e21c] [791ac603] [18f2cdab]
+- Partially implemented [`sycl_ext_intel_fpga_kernel_interface_properties`](doc/extensions/proposed/sycl_ext_intel_fpga_kernel_interface_properties.asciidoc) extension [c5d6b89c]
+- Added support for `sycl::ext::oneapi::experimental::if_architecture_is` to 
+ESIMD kernels [7a811191]
+- Implemented `sycl::opencl::cl_*` aliases [e98d732406cc]
+- Implemented `annotated_arg` headers from [`sycl_ext_oneapi_annotated_arg`](doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc) extension [1649fe46]
+- Added `bfloat16` support to imf libdevice functions [378678a3] [e85b51fc] 
+[25a3813b] [2b00cf95]
+- Added initialization of `sycl::ext::oneapi::experimental::device_global` 
```

Comment:
Tagging @steffenlarsen to review this particular line

Reply:
Since `device_global` isn't considered supported yet, I am reluctant to have this sort of wording in the release notes, at least without something specifying that exactly.